string cart_id
float64 distance
string direction
---
geometry_msgs/Pose dock_pose

